clients/egui/src/account/tree/mod.rs

```diff
@@ -45,33 +45,30 @@ impl FileTree {
     }
 
     pub fn show(&mut self, ui: &mut egui::Ui) -> NodeResponse {
-        ui.vertical(|ui| {
-            ui.spacing_mut().item_spacing = egui::vec2(0.0, 0.0);
-            let mut is_hovered = false;
-            let r = egui::Frame::none().show(ui, |ui| {
-                let result = self.root.show(ui, &mut self.state);
-                is_hovered = result.response.hovered();
-                result.inner
-            });
-
-            if self.state.is_dragging() {
-                if ui.input(|i| i.pointer.any_released()) {
-                    let maybe_pos = ui.ctx().pointer_interact_pos();
-                    self.state.dropped(maybe_pos);
-                } else {
-                    self.draw_drag_info_by_cursor(ui);
-                }
-            } else if is_hovered && ui.input(|i| i.pointer.primary_down()) {
-                // todo(steve): prep drag only if a file is clicked
-                self.state.dnd.is_primary_down = true;
-                if ui.input(|i| i.pointer.is_moving()) {
-                    self.state.dnd.has_moved = true;
-                }
+        ui.spacing_mut().item_spacing = egui::vec2(0.0, 0.0);
```

Comment:
Feels like un un-related change snuck in here?

Reply:
no that change is intentional, got rid of the `ui.vertical` because it's not needed any more and expanded the tree to take full height. 

without changes:
![image](https://github.com/lockbook/lockbook/assets/66345861/9638fde8-86c1-418a-b628-7744b2213059)

with change:
![image](https://github.com/lockbook/lockbook/assets/66345861/11499728-5db6-4555-826d-6455a99276e6)